website/docs/tutorials/image_feature_extraction.md

```diff
@@ -0,0 +1,67 @@
+---
+id: image_feature_extraction
+title: Image Feature Extraction
+sidebar_label: Image Feature Extraction
+---
+
+In this tutorial, we will go through the step-by-step process of generating image features with FasterRCNN feature extractors. MMF provides utility scripts for running image feature extractions using different models (Faster RCNN with X-101 backbones and X-152 backbones). For example: `tools/scripts/features/extract_features_vmb.py`. The script allows you to optionally parallelize the feature extraction execution. 
+
+Here are the steps in a nut shell:
+- Prerequisites: Install vqa-maskrcnn-benchmark
```

Comment:
The installation using `extract_features_vmb.py` doesn't work out of the box. One needs to install MMF first.

Correct set of instructions should actually do following steps:

- Create a new conda environment specifically for feature extraction so that it doesn't mess up current MMF env.
- git clone and install MMF.
- install VQA maskrcnn benchmark, making sure prerequisites are followed.